registrations/serializers.py

```diff
@@ -313,6 +313,7 @@ class JembiHelpdeskOutgoingSerializer(serializers.Serializer):
     inbound_created_on = serializers.DateTimeField()
     outbound_created_on = serializers.DateTimeField()
     inbound_channel_id = serializers.CharField(required=False, allow_blank=True)
+    message_id = serializers.CharField()
```

Comment:
Nothing to change here, but we just need to remember to update CasePro to send this extra field.

Reply:
so I need to remove this?

Reply:
No, nothing needs to change in this PR. We need to change CasePro to send this field to us, because currently CasePro isn't sending us the message ID.

Reply:
okay noted